app/lib/frontend/templates.dart

```diff
@@ -9,26 +9,14 @@ import 'dart:io';
 import 'dart:math';
 
 import 'package:gcloud/service_scope.dart' as ss;
-import 'package:markdown/markdown.dart';
 import 'package:mustache/mustache.dart' as mustache;
 
+import '../shared/markdown.dart';
 import '../shared/mock_scores.dart';
 
 import 'models.dart';
 import 'search_service.dart' show CseTokens, SearchQuery, SearchResultPage;
 
-final List<BlockSyntax> _blockSyntaxes =
-    ExtensionSet.gitHub.blockSyntaxes.toList();
-
-final List<InlineSyntax> _inlineSyntaxes = ExtensionSet.gitHub.inlineSyntaxes
-    .where((s) => s is! InlineHtmlSyntax)
```

Comment:
This was here in order to prevent 3-rd party html on pub.dartlang.org. Why did you remove it?